tests/README.md

```diff
@@ -44,15 +44,14 @@ which the postgres provider can connect. The server will need to have postgis
 support enabled.
 By default the test uses the following connection options:
     dbname='qgis_test'
-    host=localhost
-    port=5432
-    user='postgres'
-    password='postgres'
 
 If this does not match your setup you can set the environment variable
-QGIS_PGTEST_DB to the desired connection string.
+`QGIS_PGTEST_DB` to the desired database name.
```

Comment:
Note that `QGIS_PGTEST_DB` can contain more than just the _database name_. It can be a complete _connection string_ with all allowed options.